src/test/java/io/github/mfoo/libyear/LibYearMojoTest.java

```diff
@@ -1212,6 +1216,74 @@ public void projectExceedsMaxLibYearsAndShouldFailTheBuild() throws Exception {
                                 l.contains("This module exceeds the maximum" + " dependency age of 0.1 libyears")));
     }
 
+    @Test
+    public void reportFileGenerated(@TempDir Path tempDir) throws Exception {
+        Path reportFile = tempDir.resolve("libyear_testreport.txt");
```

Comment:
Currently the plugin doesn't sit any of the libyear findings to a file. What's the motivation for adding it here? To me it feels like something that should be everywhere or nowhere, I.e we should implement it everywhere else that we're logging currently.

Reply:
We would use the plugin for the asynchronous builds via Jenkins. Analyzing the log files is tedious; We need a file in which we can look up the results at any time and which we could also send to other project participants. And yes, I also think this can be useful for all expenses. Control is possible via a configuration parameter: either all results are written to a file or to the log output.